### PR TITLE
DCS-1450 Adding helper method for testing setting of cookies in unit tests

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/routes/__testutils/requestTestUtils.ts [hunk 1]
```diff
@@ -67,3 +68,14 @@ export const mockResponse = ({ locals = { context: {}, user: {} } }: ResponsePar
   } as unknown as jest.Mocked<Response>)
 
 export const mockNext = (): NextFunction => jest.fn()
+
+export const expectSettingCookie = (res: superAgent.Response, cookieName: string) => {
+  const [cookie] = res.header['set-cookie']
+  const [, name, value] = cookie.match(/^(.*?)=(.*?);/)
```
Comment (louisenorris) [inline server/routes/__testutils/requestTestUtils.ts:74]: not sure if this matters but is there an extra comma here?

Comment (andrewrlee) [reply server/routes/__testutils/requestTestUtils.ts:74]: that just means ignore the first value - I'd prefer to set it to a placeholder `_` or something but eslint moans about that.

Comment (andrewrlee) [reply server/routes/__testutils/requestTestUtils.ts:74]: `_` doesn't work if you need to do it twice in the same block (in other languages it's a language feature)

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**